src/components/TechnicalInfoContent/TechnicalInfoContent.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Button, Grid, List, ListItem, Typography } from '@mui/material';
+import ConfigService from '../../services/ConfigService';
+import { useSolutionData } from '../../state/hooks/SolutionHooks';
+import { useTranslation } from 'react-i18next';
+
+export const TechnicalInfoContent = () => {
+  const { t } = useTranslation();
+  const solutionData = useSolutionData();
+  const solutionName = solutionData?.name ?? '';
+  const solutionDescription = solutionData?.description ?? '';
+
+  const technicalInformation = [
+    {
+      label: t('genericcomponent.dialog.technicalInfo.webAppVersion', 'Webapp version'),
+      content: ConfigService.getParameterValue('APP_VERSION'),
+    },
+    {
+      label: t('genericcomponent.dialog.technicalInfo.apiVersion', 'API version'),
+      // TODO: add api version when available
+      content: null,
+    },
+    {
+      label: t('genericcomponent.dialog.technicalInfo.repositoryName', 'Container registry repository name'),
+      content: solutionData?.repository,
+    },
+    {
+      label: t('genericcomponent.dialog.technicalInfo.repositoryTag', 'Container registry repository tag'),
+      // TODO: add tag when available
+      content: null,
+    },
+    {
+      label: t('genericcomponent.dialog.technicalInfo.simulatorVersion', 'Simulator version'),
+      content: solutionData?.version,
+    },
+    {
+      label: t('genericcomponent.dialog.technicalInfo.sdkVersion', 'SDK version'),
+      content: solutionData?.sdkVersion,
+    },
+  ];
+  const copyInfoToClipboard = () => {
+    navigator.clipboard.writeText(
+      solutionName +
+        '\n' +
+        solutionDescription +
+        '\n' +
+        technicalInformation.map((infoItem) => `${infoItem.label}: ${infoItem.content}`).join('\n')
```

Comment:
We should probably filter on `content != null` here too (for example if a Solution doesn't define the `sdkVersion` parameter)

Reply:
it's one of my questions to Emma, my first idea was to filter it but than I thought that maybe the whole list could be useful for support as it explicitly states which information the user doesn't have :slightly_smiling_face:

Reply:
You're right, maybe it can help to be aware of missing parameters :slightly_smiling_face: